static/js/compose.js

```diff
@@ -118,6 +118,23 @@ exports.update_stream_button_for_stream = function () {
     update_stream_button(text, title);
 };
 
+function update_conversation_button(btn_text, title) {
+    $("#left_bar_compose_private_button_big").text(btn_text);
+    $("#left_bar_compose_private_button_big").prop("title", title);
+}
+
+exports.update_conversation_button_for_private = function () {
+    var text = i18n.t("New conversation");
+    var title = text + " (c)";
+    update_conversation_button(text, title);
+};
+
+exports.update_conversation_button_for_stream = function () {
+    var text = i18n.t("New private message");
+    var title = text + " (c)";
+    update_conversation_button(text, title);
+};
```

Comment:
Should we just rename the  `update_strema_button_for` to `update_closed_compose_buttons_for`  and have the existing functions handle both these functions?  That seems cleaner than adding a second function we need to call from all the same places.

Reply:
yeah...that's cleaner. I am working on the changes.